employee-rostering-frontend/src/util/BookmarkableUtils.test.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as bookmarkableUtils from './BookmarkableUtils';
+import { getRouterProps } from './BookmarkableTestUtils';
+
+// Ugly workaround for having no way to wait for all requestAnimationFrame to finish
+const flushPromisesCallback = (cb: () => void) =>
+  window.requestAnimationFrame(() => new Promise(setImmediate).then(cb));
+describe('Bookmarkable Utils', () => {
+  beforeAll(() => {
+    jest.unmock("util/BookmarkableUtils");
```

Comment:
I may be wrong. The intention here to cancel mocking of `BookmarkableUtils` just for this test makes sense. But maybe it doesn't work as expected... @Christopher-Chianelli can you investigate and shed some light on the coverage problem?

Reply:
I will investigate

Reply:
Threw errors on first lines in BookmarkableUtils; can confirm it uses mock instead of actual